### PR TITLE
Add lcjs

### PR DESCRIPTION
**Pull request by niiloArction**
Checklist:

- [x] I've read and understood [Contributing Guidelines](CONTRIBUTING.md).
- [x] I've added the new resource at the end of its section.
- [x] This resource is out there for a while, and actively maintained.
- [x] This resource is popular enough and has at least a few hundred stars on GitHub.

---

Hello, this is Niilo from the lightning chart team.
We'd much appreciate if our product could be appended to the short mention of "great commercial data visualization tools" in your GitHub page.

Currently the line is
> There're also some great commercial libraries, like amchart, anychart, plotly, and highchart.

This pull request changes it to 
> There're also some great commercial libraries, like amchart, anychart, plotly, highchart, and lightning chart.

Lightning chart is a relatively new name in web charts so perhaps it is not yet as widely known as some of the others there, but we have no doubt that this will be changing.

While it is not open-source, we do support non-commercial usage and trials for free.

Thanks for your co-operation.

### CODE REVIEW

**Comment by gokaygurcan** [comment]
I don't think we have a rule to be an open source project, so it should be fine. My only concern was to find an easy way to measure its popularity. 

On [GitHub](https://github.com/Arction), it shows 1-2 stars on each repository, but since it's closed-sourced, that makes a bit of sense. Then I followed your instructions on [docs](https://www.arction.com/lightningchart-js-api-documentation/v3.0.1/index.html#1-install-from-npm-and-use-a-bundler), it says install `@arction/lcjs`. When I check that package on [npm](https://www.npmjs.com/package/@arction/lcjs) it says around 200 weekly downloads. 

Basically, I'm having a hard time to verify your claim here. Can you please explain why do you think this repository is popular enough and should be in the list?

**Comment by Nipatsku** [comment]
Hi, thanks for your reply.

As you mentioned we're on `npm` since more than 2 years ago with consistent downloads.

LightningChart JS also has its own "community support page" in [Stack Overflow with the "lightningchart" tag](https://stackoverflow.com/questions/tagged/lightningchart), there you can see constant life in form of user questions and replies from the devs.

We host open-source comparisons of different JavaScript data visualization libraries, for example [this](https://github.com/Arction/javascript-charts-performance-comparison).

And finally [here](https://www.arction.com/lightningchart-js/) is our commercial product web site.

**Comment by gokaygurcan** [comment]
Fair enough, let's go 🚀
